From 82691c53c7d84f79116b3f94cbe5fd7d7006d5e4 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Tue, 23 Aug 2016 09:43:14 +0200
Subject: [PATCH 1/1] remove const qualifier and fix crash with wsrep
 initialization

Compiling MariaDB with GCC 6.x makes it crash in wsrep/galera
initialization. This fixes the crash.

Signed-off-by: Christian Hesse <mail@eworm.de>
---
 sql/wsrep_mysqld.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sql/wsrep_mysqld.cc b/sql/wsrep_mysqld.cc
index 823a2a7..8760a11 100644
--- a/sql/wsrep_mysqld.cc
+++ b/sql/wsrep_mysqld.cc
@@ -169,7 +169,7 @@ static PSI_file_info wsrep_files[]=
 
 my_bool wsrep_inited                   = 0; // initialized ?
 
-static const wsrep_uuid_t cluster_uuid = WSREP_UUID_UNDEFINED;
+static wsrep_uuid_t cluster_uuid = WSREP_UUID_UNDEFINED;
 static char         cluster_uuid_str[40]= { 0, };
 static const char*  cluster_status_str[WSREP_VIEW_MAX] =
 {
-- 
2.9.3

